index.js

```diff
@@ -435,5 +435,19 @@ export const mentorshipTeam = [
 			},
 		],
 	},
+	{
+        name: 'Phuong Le',
+        title: 'Student @ Denison University',
+        hobbies: 'films, novels, pop ballad',
+        avatar:
+            'https://drive.google.com/file/d/1aMiVwma-maKyouo6cHLHR_TL_VYEKdMI/view?usp=drive_link',
```

Comment:
let's change to this link `https://res.cloudinary.com/khoa165/image/upload/v1711262168/viettech/phuongle.jpg`